app/utils/extensions/string.ts

```diff
@@ -10,6 +10,12 @@ interface String {
      * @example " This is a   string ".clearWhitespace(); //=> "Thisisastring"
      */
     clearWhitespace();
+
+    /**
+     * Add padding to the start of a string so that is in N characters long.
+     * '1'.padStart(5, '0') = 00001
+     */
+    padStart(maxLength: number, padString: string);
```

Comment:
in the `padStart` specs `padString` is optional, should probably change it. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart